src/pipelinedefinition.cpp

```diff
@@ -1288,7 +1276,7 @@ const NodeInfo& PipelineDefinition::findNodeByName(const std::string& name) cons
     });
 }
 
-Shape PipelineDefinition::getNodeGatherShape(const NodeInfo& info) const {
+Shape PipelineDefinition::getNodeGatherShape(const NodeInfo& info) {
```

Comment:
What has changed that forced us to remove `const` here?

Reply:
Inside that function we call `getCustomNodeMetadata`. One of its arguments was void*, but it is changed to shared_ptr&. Trying to compile with const keyword causes compilation error.

Reply:
Could it be `const shared_ptr&` then? Or `void*` as you mentioned earlier?

Reply:
const shared_ptr& solved the problem